### ISSUE TITLE
Map does not load

### ISSUE THREAD

**Issue by ch000** [FogosPT/fogosmobile#174]
**Describe the bug**
When i open the app, i only see map icons, but no map. Instead of the map see a black background

**To Reproduce**
1. Download App
2. Open App

**Expected behavior**
To see a map of portugal

**Screenshots**
![photo_2023-07-17 17 17 42](https://github.com/FogosPT/fogosmobile/assets/2071241/3ec37de3-7295-4484-87c5-12b719f76a6a)
 
**Smartphone (please complete the following information):**
 - Device: Pixel 4a
 - OS: Android 13

**Comment by kikosgc** [comment]
+1 Tested both with Android and iOS and getting same results (black map).
Devices tested: 
- Apple Iphone 14 Pro with iOS 16.6
- Samsung S22 with Android 13
- Samsung S10 with Android 12
- Samsung S10e with Android 12

Tested and working OK on a Apple Iphone 13 Pro Max with iOS 16.6

**Comment by ner00** [comment]
This happened to me after I clear the app's cache, so whatever map it was using before was probably completely offline and can no longer be downloaded, either that or some API key was changed or revoked.

**Comment by ner00** [comment]
@tomahock 
@guivazcabral
@miguelpruivo 
@jcalado 
@Vanethos

**Comment by ner00** [comment]
Há um problema com o _token_ do Mapbox `{"message":"Not Authorized - Invalid Token"}`:

```
08-10 10:25:18.808 10481 10637 D Mbgl-HttpRequest: [HTTP] Request with response = 401: Unauthorized
08-10 10:25:18.811 10481 10598 D Mbgl-HttpRequest: [HTTP] This request was cancelled (https://api.mapbox.com/styles/v1/fogospt/cjgppvcdp00aa2spjclz9sjst?access_token=pk.bWUMn7KcTkkdgWqOtu6h3hsLvqct9PDDc9vfMioS3RrLL2eNCnwKqpmOPpdioVB.advBLHs3QFLIPiUgOtLt0o&sku=100ll4xasme857be74c6fe94e388fc4decc90e52d6f). This is expected for tiles that were being prefetched but are no longer needed for the map to render.
08-10 10:25:18.815 10481 10633 E Mbgl    : {.tomahock.fogos}[Setup]: loading style failed: HTTP status code 401
```

...mas não entendo porque apenas dá problema para novas instalações.

logcat dump processo `com.tomahock.fogos`
```
08-10 10:25:17.728 10481 10481 I libc    : SetHeapTaggingLevel: tag level set to 0
08-10 10:25:17.736 10481 10481 I .tomahock.fogos: Using CollectorTypeCC GC.
08-10 10:25:17.742 10481 10481 E .tomahock.fogos: Not starting debugger since process cannot load the jdwp agent.
08-10 10:25:17.747 10481 10481 I MessageMonitor: Load libmiui_runtime
08-10 10:25:17.775 10481 10481 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10407; state: DISABLED
08-10 10:25:17.780 10481 10481 D ApplicationLoaders: Returning zygote-cached class loader: /system/framework/android.test.base.jar
08-10 10:25:17.789 10481 10500 D AppScoutStateMachine: 10481-ScoutStateMachinecreated
08-10 10:25:17.790 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.arm64_v8a.dm': No such file or directory
08-10 10:25:17.790 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.arm64_v8a.dm': No such file or directory
08-10 10:25:17.790 10481 10481 W .tomahock.fogos: Entry not found
08-10 10:25:17.791 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.en.dm': No such file or directory
08-10 10:25:17.791 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.en.dm': No such file or directory
08-10 10:25:17.791 10481 10481 W .tomahock.fogos: Entry not found
08-10 10:25:17.791 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.pt.dm': No such file or directory
08-10 10:25:17.791 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.pt.dm': No such file or directory
08-10 10:25:17.791 10481 10481 W .tomahock.fogos: Entry not found
08-10 10:25:17.791 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.xxhdpi.dm': No such file or directory
08-10 10:25:17.791 10481 10481 W ziparchive: Unable to open '/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.xxhdpi.dm': No such file or directory
08-10 10:25:17.791 10481 10481 W .tomahock.fogos: Entry not found
08-10 10:25:17.792 10481 10481 D nativeloader: Configuring classloader-namespace for other apk /data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/base.apk:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.arm64_v8a.apk:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.en.apk:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.pt.apk:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.xxhdpi.apk. target_sdk_version=29, uses_libraries=, library_path=/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/lib/arm64:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/base.apk!/lib/arm64-v8a:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.arm64_v8a.apk!/lib/arm64-v8a:/data/app/~~az_n0B2PYKHdjU0Evycxbg==/com.tomahock.fogos-uAEeV2qgTBSsIx-Ek8aI8w==/split_config.en.
08-10 10:25:17.792 10481 10481 D nativeloader: Extending system_exposed_libraries: libbinauralrenderer_wrapper.qti.so:libhoaeffects.qti.so:libQOC.qti.so:libmisys_jni.xiaomi.so:libupdateprof.qti.so:libthermalclient.qti.so:libQOC.qti.so:libdiag_system.qti.so:libqape.qti.so:liblistenjni.qti.so
08-10 10:25:17.794 10481 10481 W .tomahock.fogos: JIT profile information will not be recorded: profile file does not exist.
08-10 10:25:17.794 10481 10481 W .tomahock.fogos: JIT profile information will not be recorded: profile file does not exist.
08-10 10:25:17.794 10481 10481 W .tomahock.fogos: JIT profile information will not be recorded: profile file does not exist.
08-10 10:25:17.794 10481 10481 W .tomahock.fogos: JIT profile information will not be recorded: profile file does not exist.
08-10 10:25:17.799 10481 10481 I Perf    : Connecting to perf service.
08-10 10:25:17.807 10481 10481 V GraphicsEnvironment: ANGLE Developer option for 'com.tomahock.fogos' set to: 'default'
08-10 10:25:17.807 10481 10481 V GraphicsEnvironment: ANGLE GameManagerService for com.tomahock.fogos: false
08-10 10:25:17.807 10481 10481 V GraphicsEnvironment: Updatable production driver is not supported on the device.
08-10 10:25:17.810 10481 10481 I ForceDarkHelperStubImpl: initialize for com.tomahock.fogos , ForceDarkOrigin
08-10 10:25:17.811 10481 10481 D .tomahock.fogos: JNI_OnLoad success
08-10 10:25:17.812 10481 10481 I MiuiForceDarkConfig: setConfig density:2.750000, mainRule:0, secondaryRule:0, tertiaryRule:0
08-10 10:25:17.812 10481 10481 D NetworkSecurityConfig: No Network Security Config specified, using platform default
08-10 10:25:17.812 10481 10481 D NetworkSecurityConfig: No Network Security Config specified, using platform default
08-10 10:25:17.814 10481 10481 D CompatibilityChangeReporter: Compat change id reported: 183155436; UID 10407; state: DISABLED
08-10 10:25:17.824 10481 10481 I FirebaseInitProvider: FirebaseApp initialization successful
08-10 10:25:17.845 10481 10481 D MiuiMultiWindowAdapter: MiuiMultiWindowAdapter::getFreeformVideoWhiteListInSystem::LIST_ABOUT_SUPPORT_LANDSCAPE_VIDEO = [com.hunantv.imgo.activity, com.tencent.qqlive, com.qiyi.video, com.hunantv.imgo.activity.inter, com.tencent.qqlivei18n, com.iqiyi.i18n, tv.danmaku.bili]
08-10 10:25:17.854 10481 10481 W .tomahock.fogos: type=1400 audit(0.0:1704): avc: denied { read } for name="u:object_r:vendor_displayfeature_prop:s0" dev="tmpfs" ino=12262 scontext=u:r:untrusted_app_29:s0:c151,c257,c512,c768 tcontext=u:object_r:vendor_displayfeature_prop:s0 tclass=file permissive=0 app=com.tomahock.fogos
08-10 10:25:17.861 10481 10481 W libc    : Access denied finding property "ro.vendor.df.effect.conflict"
08-10 10:25:17.863 10481 10517 I FA      : App measurement is starting up, version: 16250
08-10 10:25:17.863 10481 10517 I FA      : To enable debug logging run: adb shell setprop log.tag.FA VERBOSE
08-10 10:25:17.863 10481 10517 I FA      : To enable faster debug mode event logging run:
08-10 10:25:17.863 10481 10517 I FA      :   adb shell setprop debug.firebase.analytics.app com.tomahock.fogos
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: QUALCOMM build                   : 8382da08aa, I139f36ad46
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Build Date                       : 02/18/22
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.35.01.11
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Local Branch                     : 
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Remote Branch                    : 
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Remote Branch                    : 
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Reconstruct Branch               : 
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Build Config                     : S P 10.0.7 AArch64
08-10 10:25:17.874 10481 10481 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
08-10 10:25:17.878 10481 10481 I AdrenoGLES-0: PFP: 0x016dc094, ME: 0x00000000
08-10 10:25:17.893 10481 10532 W libc    : Access denied finding property "vendor.migl.debug"
08-10 10:25:17.931 10481 10481 D IS_CTS_MODE: false
08-10 10:25:17.931 10481 10481 D MULTI_WINDOW_ENABLED: false
08-10 10:25:17.932 10481 10481 D DecorView[]: getWindowModeFromSystem  windowmode is 1
08-10 10:25:17.938 10481 10517 I FA      : Tag Manager is not found and thus will not be used
08-10 10:25:17.953 10481 10481 D DecorView[]: getWindowModeFromSystem  windowmode is 1
08-10 10:25:17.953 10481 10481 D DecorView[]: updateDecorCaptionStatus windowMode is 1
08-10 10:25:17.992 10481 10481 D SurfaceView: UPDATE null, mIsCastMode = false
08-10 10:25:18.004 10481 10526 E libEGL  : pre_cache appList: ,,
08-10 10:25:18.159 10481 10481 E SurfaceSyncer: Failed to find sync for id=0
08-10 10:25:18.161 10481 10526 D .tomahock.fogos: MiuiProcessManagerServiceStub setSchedFifo
08-10 10:25:18.161 10481 10526 I MiuiProcessManagerImpl: setSchedFifo pid:10481, mode:3
08-10 10:25:18.184 10481 10526 E LB      : fail to open file: No such file or directory
08-10 10:25:18.187 10481 10526 W Parcel  : Expecting binder but got null!
08-10 10:25:18.188 10481 10530 I flutter : SharedPreferencesManager: Preferences loaded.
08-10 10:25:18.192 10481 10481 D SurfaceView: UPDATE Surface(name=SurfaceView[com.tomahock.fogos/com.example.fogosmobile.MainActivity])/@0xa6dd16c, mIsProjectionMode = false
08-10 10:25:18.192 10481 10481 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-10 10:25:18.192 10481 10481 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-10 10:25:18.192 10481 10481 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-10 10:25:18.208 10481 10481 D DecorView[]: onWindowFocusChanged hasWindowFocus true
08-10 10:25:18.208 10481 10481 I HandWritingStubImpl: refreshLastKeyboardType: 1
08-10 10:25:18.208 10481 10481 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-10 10:25:18.212 10481 10481 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-10 10:25:18.219 10481 10481 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-10 10:25:18.219 10481 10481 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-10 10:25:18.227 10481 10530 I flutter : token: 9Yz7df6M4gg:emT52qNrISuRayJskvXuDNv1uHRo2WEIgSKeSD6Nr3qEnGH4OePqxULSkbsnW1jwj2fVmld5T1TG9CqbwyUWjQ2vSJQ5tVkJZiG6mA60k5Y2VPWTCtjBVxYfm1Ag4x0DxhXPcSlUCP2y
08-10 10:25:18.285 10481 10481 E r       : setCameraTargetBounds is supported only after map initiated.
08-10 10:25:18.288 10481 10594 W CrashReporter: Root directory doesn't exist
08-10 10:25:18.323 10481 10606 D TrafficStats: tagSocket(156) with statsTag=0xffffffff, statsUid=-1
08-10 10:25:18.355 10481 10481 E TelemetryUtils: java.lang.SecurityException: getDataNetworkTypeForSubscriber
08-10 10:25:18.378 10481 10481 D IS_CTS_MODE: false
08-10 10:25:18.378 10481 10481 D MULTI_WINDOW_ENABLED: false
08-10 10:25:18.415 10481 10481 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-10 10:25:18.419 10481 10481 V Mbgl-ConnectivityReceiver: connected - true
08-10 10:25:18.440 10481 10596 I Mbgl-EGLConfigChooser: In emulator: false
08-10 10:25:18.444 10481 10526 W Parcel  : Expecting binder but got null!
08-10 10:25:18.451 10481 10481 D DecorView[]: getWindowModeFromSystem  windowmode is 1
08-10 10:25:18.451 10481 10481 D DecorView[]: updateDecorCaptionStatus windowMode is 1
08-10 10:25:18.454 10481 10530 I flutter : Request to https://fogos.pt/lightnings performed with success (200).
08-10 10:25:18.454 10481 10530 I flutter : type '_GrowableList<dynamic>' is not a subtype of type 'Map<String, dynamic>' in type cast
08-10 10:25:18.454 10481 10530 I flutter : #0      _createLightningStrikes.<anonymous closure> (package:fogosmobile/middleware/lightnings_middleware.dart:26)
08-10 10:25:18.454 10481 10530 I flutter : <asynchronous suspension>
08-10 10:25:18.478 10481 10633 I Mbgl    : {Thread-6}[General]: GPU Identifier: Adreno (TM) 642
08-10 10:25:18.502 10481 10530 I flutter : Request to https://fogos.pt/js/dico.json performed with success (200).
08-10 10:25:18.548 10481 10637 D TrafficStats: tagSocket(247) with statsTag=0xffffffff, statsUid=-1
08-10 10:25:18.621 10481 10530 I flutter : Request to https://fogos.pt/v1/viirs performed with success (200).
08-10 10:25:18.645 10481 10530 I flutter : Request to https://fogos.pt/v1/viirs performed with success (200).
08-10 10:25:18.679 10481 10530 I flutter : Request to https://fogos.pt/v1/modis performed with success (200).
08-10 10:25:18.698 10481 10530 I flutter : Request to https://fogos.pt/v1/modis performed with success (200).
08-10 10:25:18.730 10481 10530 I flutter : Request to https://api-lb.fogos.pt/new/fires performed with success (200).
08-10 10:25:18.743 10481 10530 I flutter : Request to https://api-lb.fogos.pt/new/fires performed with success (200).
08-10 10:25:18.808 10481 10637 D Mbgl-HttpRequest: [HTTP] Request with response = 401: Unauthorized
08-10 10:25:18.811 10481 10598 D Mbgl-HttpRequest: [HTTP] This request was cancelled (https://api.mapbox.com/styles/v1/fogospt/cjgppvcdp00aa2spjclz9sjst?access_token=pk.bWUMn7KcTkkdgWqOtu6h3hsLvqct9PDDc9vfMioS3RrLL2eNCnwKqpmOPpdioVB.advBLHs3QFLIPiUgOtLt0o&sku=100ll4xasme857be74c6fe94e388fc4decc90e52d6f). This is expected for tiles that were being prefetched but are no longer needed for the map to render.
08-10 10:25:18.815 10481 10633 E Mbgl    : {.tomahock.fogos}[Setup]: loading style failed: HTTP status code 401
08-10 10:25:20.044 10481 10481 W MirrorManager: this model don't Support
08-10 10:25:20.178 10481 10530 I flutter : Request to https://fogos.pt/js/dico.json performed with success (200).
08-10 10:25:20.179 10481 10530 I flutter : Request to https://fogos.pt/lightnings performed with success (200).
08-10 10:25:20.179 10481 10530 I flutter : type '_GrowableList<dynamic>' is not a subtype of type 'Map<String, dynamic>' in type cast
08-10 10:25:20.179 10481 10530 I flutter : #0      _createLightningStrikes.<anonymous closure> (package:fogosmobile/middleware/lightnings_middleware.dart:26)
08-10 10:25:20.179 10481 10530 I flutter : <asynchronous suspension>
08-10 10:25:20.237 10481 10530 I flutter : Request to https://fogos.pt/v1/modis performed with success (200).
08-10 10:25:20.260 10481 10530 I flutter : Request to https://api-lb.fogos.pt/new/fires performed with success (200).
08-10 10:25:20.268 10481 10530 I flutter : Request to https://fogos.pt/v1/viirs performed with success (200).
08-10 10:25:22.257 10481 10530 I flutter : SharedPreferencesManager: Saving value [1] for key [pref-satellite].
08-10 10:25:22.333 10481 10637 D Mbgl-HttpRequest: [HTTP] Request with response = 401: Unauthorized
08-10 10:25:22.336 10481 10598 D Mbgl-HttpRequest: [HTTP] This request was cancelled (https://api.mapbox.com/styles/v1/fogospt/cjksgciqsctfg2rp9x9uyh37g?access_token=pk.bWUMn7KcTkkdgWqOtu6h3hsLvqct9PDDc9vfMioS3RrLL2eNCnwKqpmOPpdioVB.advBLHs3QFLIPiUgOtLt0o&sku=100ll4xasme857be74c6fe94e388fc4decc90e52d6f). This is expected for tiles that were being prefetched but are no longer needed for the map to render.
08-10 10:25:22.336 10481 10633 E Mbgl    : {.tomahock.fogos}[Setup]: loading style failed: HTTP status code 401
08-10 10:25:23.850 10481 10530 I flutter : SharedPreferencesManager: Saving value [0] for key [pref-satellite].
08-10 10:25:23.929 10481 10637 D Mbgl-HttpRequest: [HTTP] Request with response = 401: Unauthorized
08-10 10:25:23.930 10481 10598 D Mbgl-HttpRequest: [HTTP] This request was cancelled (https://api.mapbox.com/styles/v1/fogospt/cjgppvcdp00aa2spjclz9sjst?access_token=pk.bWUMn7KcTkkdgWqOtu6h3hsLvqct9PDDc9vfMioS3RrLL2eNCnwKqpmOPpdioVB.advBLHs3QFLIPiUgOtLt0o&sku=100ll4xasme857be74c6fe94e388fc4decc90e52d6f). This is expected for tiles that were being prefetched but are no longer needed for the map to render.
08-10 10:25:23.935 10481 10633 E Mbgl    : {.tomahock.fogos}[Setup]: loading style failed: HTTP status code 401
08-10 10:25:27.832 10481 10496 I .tomahock.fogos: Background concurrent copying GC freed 474216(20MB) AllocSpace objects, 44(1828KB) LOS objects, 75% free, 3721KB/14MB, paused 72us,15.504ms total 92.412ms
08-10 10:25:28.879 10481 10526 W RenderInspector: DequeueBuffer time out on , count=1, avg=22 ms, max=22 ms.
08-10 10:26:05.395 10481 10481 D DecorView[]: onWindowFocusChanged hasWindowFocus false
08-10 10:44:47.858 10481 10650 D AppScoutStateMachine: 10481-ScoutStateMachinecreated
08-10 10:44:47.908 10481 10481 D SurfaceView: UPDATE Surface(name=SurfaceView[com.tomahock.fogos/com.example.fogosmobile.MainActivity])/@0x6f02586, mIsProjectionMode = false
08-10 10:44:47.909 10481 10481 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-10 10:44:47.909 10481 10481 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-10 10:44:47.910 10481 10481 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-10 10:44:47.911 10481 10481 E SurfaceSyncer: Failed to find sync for id=0
08-10 10:44:47.911 10481 10481 E SurfaceSyncer: Failed to find sync for id=1
08-10 10:44:47.918 10481 10481 V Mbgl-ConnectivityReceiver: connected - true
08-10 10:44:48.077 10481 10481 D IS_CTS_MODE: false
08-10 10:44:48.077 10481 10481 D MULTI_WINDOW_ENABLED: false
08-10 10:44:48.078 10481 10530 I flutter : Request to https://fogos.pt/lightnings performed with success (200).
08-10 10:44:48.078 10481 10530 I flutter : type '_GrowableList<dynamic>' is not a subtype of type 'Map<String, dynamic>' in type cast
08-10 10:44:48.078 10481 10530 I flutter : #0      _createLightningStrikes.<anonymous closure> (package:fogosmobile/middleware/lightnings_middleware.dart:26)
08-10 10:44:48.078 10481 10530 I flutter : <asynchronous suspension>
08-10 10:44:48.140 10481 10530 I flutter : Request to https://fogos.pt/v1/viirs performed with success (200).
08-10 10:44:48.141 10481 10526 W Parcel  : Expecting binder but got null!
08-10 10:44:48.147 10481 10481 D DecorView[]: onWindowFocusChanged hasWindowFocus true
08-10 10:44:48.147 10481 10481 I HandWritingStubImpl: refreshLastKeyboardType: 1
08-10 10:44:48.147 10481 10481 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-10 10:44:48.161 10481 10530 I flutter : Request to https://fogos.pt/v1/modis performed with success (200).
08-10 10:44:48.165 10481 10481 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-10 10:44:48.200 10481 10481 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-10 10:44:48.240 10481 10530 I flutter : Request to https://api-lb.fogos.pt/new/fires performed with success (200).
08-10 10:44:48.314 10481 10481 D IS_CTS_MODE: false
08-10 10:44:48.315 10481 10481 D MULTI_WINDOW_ENABLED: false
08-10 10:44:48.408 10481 10526 W Parcel  : Expecting binder but got null!
08-10 10:44:49.776 10481 10481 D DecorView[]: onWindowFocusChanged hasWindowFocus false
```

**Comment by ner00** [comment]
Existe atividade recente, talvez em breve tenhamos uma surpresa positiva.
https://github.com/FogosPT/fogosmobile/tree/vega

**Comment by tomahock** [comment]
Resolvido!
